statistics/cmsketch.go

```diff
@@ -126,10 +278,26 @@ func CMSketchToProto(c *CMSketch) *tipb.CMSketch {
 			protoSketch.Rows[i].Counters[j] = c.table[i][j]
 		}
 	}
+	if c.topnindex != nil {
+		protoSketch.TopN = make([]*tipb.CMSketchTopN, len(c.topnindex))
+		seq := uint32(0)
+		for _, v1 := range c.topnindex {
+			for _, v := range v1 {
+				h1, h2 := murmur3.Sum128(v.data)
+				for i := range c.table {
+					j := (h1 + h2*uint64(i)) % uint64(c.width)
+					protoSketch.Rows[i].Counters[j] += uint32(v.count)
```

Comment:
It may exceed uint32, so it is better not to store it in counters.

Reply:
If recovered with correct top n data, this will not make any different.
If not recovered with top n data, this cmsketch is the same as a cmsketch without top n fed by same data.

Reply:
Origin sketch may have problems when there are values occurred more than max uint32. It is better to fix them here.